pandas/_libs/groupby.pyx

```diff
@@ -770,25 +770,25 @@ def group_ohlc(floating[:, ::1] out,
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
-def group_quantile(ndarray[float64_t] out,
+def group_quantile(ndarray[float64_t, ndim=2] out,
```

Comment:
i think we name these _2d (or similar), thinking of rank of these are 2d based?

Reply:
not in this file

Reply:
kk, prob want to make a pass over the cython code to unify the names (e.g. append 1d or 2d) where appropriate